src/main/java/com/codessquad/qna/domain/Question.java

```diff
@@ -16,55 +20,95 @@
     private User writer;
 
     private String title;
+
+    @Lob
     private String contents;
-    private LocalDateTime postTime = LocalDateTime.now();
+
+    private LocalDateTime postTime;
     private LocalDateTime updatedPostTime;
+    private boolean deleted;
+
+    @OneToMany(mappedBy = "question")
+    @OrderBy("id asc")
+    @Where(clause = "deleted = false")
+    private List<Answer> answers;
+
+    protected Question() {
+    }
+
+    public Question(User writer, String title, String contents) {
+        this.writer = writer;
+        this.title = title;
+        this.contents = contents;
+        this.postTime = LocalDateTime.now();
+    }
 
     public Long getId() {
         return id;
     }
 
-    public String getWriter() {
+    public String getWriterUserId() {
         return writer.getUserId();
     }
 
-    public void setWriter(User user) {
-        this.writer = user;
-    }
-
     public String getTitle() {
         return title;
     }
 
-    public void setTitle(String title) {
-        this.title = title;
-    }
-
     public String getContents() {
         return contents;
     }
 
-    public void setContents(String contents) {
-        this.contents = contents;
-    }
-
     public String getFormattedPostTime() {
         if (updatedPostTime == null) {
             return postTime.format(DATE_TIME_FORMATTER);
         }
         return updatedPostTime.format(DATE_TIME_FORMATTER);
     }
 
+    public List<Answer> getAnswers() {
+        return answers;
+    }
+
+    public boolean isDeleted() {
+        return deleted;
+    }
+
+    public int getAnswerNum() {
+        return answers.size();
+    }
+
+    public boolean isAnswerEmpty() {
+        return answers.size() == 0;
```

Comment:
이럴때 사용할 수 있는 [이런 메소드](https://docs.oracle.com/javase/8/docs/api/java/util/List.html#isEmpty--) 가 있습니다.

```suggestion
        return answers.isEmpty();